K2Bridge/ElasticQueryTranslator.cs

```diff
@@ -67,23 +67,17 @@ public QueryData TranslateQuery(string header, string query)
                 if (elasticSearchDsl.Query.Bool != null)
                 {
                     Ensure.IsNotNull(elasticSearchDsl.Query.Bool.Must, nameof(elasticSearchDsl.Query.Bool.Must));
+                    Ensure.IsNotNull(elasticSearchDsl.Query.Bool.Filter, nameof(elasticSearchDsl.Query.Bool.Filter));
 
-                    if (elasticSearchDsl.Query.Bool.Filter.Any())
```

Comment:
I think its better to keep the 'using kql' warning. wdyt?

Reply:
Filters are not longer an indication for kql syntax as they used even in regular cases (like the date range, etc.)
Perhaps there's something else that could be used...